frontend/types/api/notifications.ts

```diff
@@ -149,6 +149,7 @@ export interface NotificationSettingsAccountDashboard {
   subscribed_chain_ids: number /* uint64 */[];
   is_incoming_transactions_subscribed: boolean;
   is_outgoing_transactions_subscribed: boolean;
+  is_erc20_token_transfers_subscribed: number /* float64 */;
```

Comment:
Maybe here we want a boolean (the next member contains the amount).
Or this member disappears and we use `< 0` to disable

Reply:
Yes it should be a boolean, thx